discord/user.py

```diff
@@ -100,13 +101,19 @@ def __ne__(self, other):
     def __hash__(self):
         return self.id >> 22
 
+    def has_flag(self, flag: UserFlags):
+        v = flag.value
+        return (self._public_flags & v) == v
+
     def _update(self, data):
         self.name = data['username']
         self.id = int(data['id'])
         self.discriminator = data['discriminator']
         self.avatar = data['avatar']
+        self._public_flags = data.get("public_flags", 0)
```

Comment:
```suggestion
        self._public_flags = data.get('public_flags', 0)
```

Single quotes are used for strings here.